model/driver/src/icon4py/model/driver/icon4py_driver.py

```diff
@@ -10,20 +10,24 @@
 import logging
 import pathlib
 import uuid
-from typing import Callable
+from typing import Callable, NamedTuple
 
 import click
 from devtools import Timer
 from gt4py.next import gtfn_cpu
 
+import icon4py.model.common.utils as imc_utils
```

Comment:
why `imc_utils`? _i_con4py _m_odel  _c_ommon? Honestly, I don't like these kind of names. Why not `utils` or `common_utils`?